### DISCUSSION TITLE
Installation in Win11 venv

### DISCUSSION THREAD

**Discussion by cnske** [GeoStat-Framework/PyKrige#276] [Q&A] [answered]
Hej I wanted to install pykrige today on my Win11 company machine using `pip install pykrige`, but my system always wants to install Microsoft C++ Build tools to install, which I can not do since I am not the admin on my machine. 
Is it possible to install pykrige without Microsoft C++ Build tools?

Usually work on Linux and Mac, Windows is a whole new world for me.

**Comment by MuellerSeb** [comment]
Hey there, I guess this is because there are no wheels for Python 3.11 and 3.12 yet on PyPI. So either you use an older version of Python in your venv, or you use pykrige from conda-forge (if you want to / can use conda), there we at least provide a Python 3.11 version.

Python 3.11 and 3.12 wheels will come with the next release (and maybe this will be the only change in the next release).

**Comment by MuellerSeb** [comment, marked as answer]
Should work now with installing Version 1.7.1
